src/creddit/client.clj

```diff
@@ -119,6 +119,12 @@
     (-> (http-get credentials (str "https://www.reddit.com/r/" subreddit "/top/.json?limit=" limit "&t=" (name time)))
         (parse-response))))
 
+(defn subreddit-hot
+  [credentials subreddit limit]
+  (if (valid-limit? limit)
+    (-> (http-get credentials (str "https://www.reddit.com/r/" subreddit "/top/.json?limit=" limit))
```

Comment:
I assume this should say `/hot` not `/top`?

Reply:
Right, overlooked that one!